### DISCUSSION TITLE
Cut & Engraved in the same file

### DISCUSSION THREAD

**Discussion by sebatiduroc** [arkypita/LaserGRBL#1359] [Q&A]
Hi,

I'm a newbee in lasercut.
First, apologize for my english...

I've a Ortur Master Laser 2 and I use LaserGrbl with windows 10.

If I use a file made with Inkscape, with some lines in Red (255,0,0) and other in Blue (0,0,255), it does not do the job I would want...
Red = cut and Blue = engraved, isn't it ?

How to code this with LaserGrbl, if it's possible ?

Thank you so much !

**Comment by StuartB4** [comment]
Not possible because LaserGRBL does'nt use layers. 
You could do it with 2 seperate files.
Load the first one and engrave, then click Append file and load the cut file.

**Reply by sebatiduroc** [reply]
Thanks. It's what i thinked...

**Reply by StuartB4** [reply]
:+1:

**Reply by Sybrand66** [reply]
Hi also a newbie. 
I have managed to do laser engraving but I just cant get it to cut. What I am I doing wrong or what do I miss. If someone can please help me. Been 4 weeks now.

**Reply by gmmanonymus111** [reply]
What type of laser do you have?
